tests/acceptance/features/split_rule.feature

```diff
@@ -10,10 +10,14 @@ Feature: Split Rule
     When make a credit card transaction
     Then a transaction must be created
     And the transaction must contain split rule
+    And the split rule must be countable
+    And the split rules count must be "2"
 
   Scenario: Create a boleto transaction with split rule
     Given a valid customer
     And valid splitRule
     When make a boleto transaction
     Then a transaction must be created
     And the transaction must contain split rule
+    And the split rule must be countable
```

Comment:
Posso remover o teste, mas acredito que melhor pecar pelo excesso do que pela falta.

Só fiz esse teste pois outros similares já ocorriam, por exemplo o teste de comportamento que verificava se o retorno era um `SplitRuleCollection` (**the transaction must contain split rule**)